### PR TITLE
Issue 9 - Support for ignoring outfitt items

### PR DESCRIPTION
**Pull request by robgha01**
This is a pull request for https://github.com/Kruithne/EasyDisenchant/issues/9
Adding a debug function using ViragDevTool if installed.
Adding support for https://wow.curseforge.com/projects/outfitter outfit items is now ignored.

### CODE REVIEW

**Comment by Kruithne** [comment]
Looking good. Could you extend this to check the Blizzard equipment sets, too? Also noted some typos.

File: EasyDisenchant.lua [hunk 1]
```diff
@@ -231,6 +238,42 @@ do
...
+					debug("Checking item: " .. vItemInfo.Link)
+					debug("bagIndex:", bagID, bagIndex)
+					debug("bagSlotIndex:", slotID, bagSlotIndex)
+					debug("itemID:", itemID, vItemInfo.Code)
+					debug("", vItemInfo)
+					
+					if bagIndex == bagID and bagSlotIndex == slotID and vItemInfo.Code == itemID then
+						debug("Check UsedInOutfit", vItemInfo.UsedInOutfit)
+
+						-- We found our item now check if its used in an outfitt.
```
Comment (Kruithne) [inline EasyDisenchant.lua]: Typo in Outfit in comment and debug message.

Comment (robgha01) [reply EasyDisenchant.lua]: I will see what i can do

**Comment by robgha01** [comment]
I have now extended it to support blizzard equipment sets and fixed some typos.
https://github.com/Kruithne/EasyDisenchant/pull/10/commits/da70e3e9a14efd5434ab10dba31333c408c7a1d0

**Comment by robgha01** [comment]
@Kruithne let me change it back, i played around a bit with playsound and forgot to change it back.
also fixed a bugg with outfitter in commit: https://github.com/Kruithne/EasyDisenchant/pull/10/commits/bcafc07465794105ce779dadfa99bf81b6c224fc

**Pull request merged**